pkg/apis/migration/v1alpha1/verify.go

```diff
@@ -0,0 +1,248 @@
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Types
+const (
+	UnhealthySourceNamespaces      = "UnhealthySourceNamespaces"
+	UnhealthyDestinationNamespaces = "UnhealthyDestinationNamespaces"
+)
+
+// Reasons
+const (
+	ErrorsDetected = "ErrorsDetected"
+)
+
+// Messages
+const (
+	UnhealthyDestination = "The destination cluster has unhealthy migrated namespaces []. See status.namespaces for details."
```

Comment:
Since the `UnhealthyNamespaces` stucture contains the full list/description, I think the message should contain the list `[]` as well.

Reply:
From what I see in the message, there is a list `[]` already. Could you please clarify?

Reply:
Meant should **not** include the `[] ` list.